server/controller.js

```diff
@@ -309,16 +367,75 @@ const handlerFunctions = {
     res.status(200).send(pantryRecipes)
   },
 
-  // get all ingredients
-
   // get all recipes
+  getAllRecipes: async (req, res) => {
+    const { pageNum } = req.params
+    const allRecipes = await Recipe.findAll({ 
+      offset: pageNum * 20, 
+      limit: 20,
+      order: [
+        ['recipeId', 'ASC']
+      ]
+    })
 
-  // get all recipe ingredients
+    console.log('allRecipes:', allRecipes)
 
-  // get all labels
+    res.status(200).send(allRecipes)
+  },
 
-  // get recipe labels
+  // get recipe ingredients by recipe id
+  getRecipeIngredientsByRecipeId: async (req, res) => {
+    const { id } = req.params
 
+    const recipeIngredients = await Ingredient.findAll({
+      include: [
+        {
+          model: RecipeIngredient,
+          where: { recipeId: id },
+          attributes: []
+        }
+      ]
+    })
+
+    res.status(200).send(recipeIngredients)
+  },
+
+  // get recipe labels by recipe id
+  getRecipeLabelsByRecipeId: async (req, res) => {
+    const { id } = req.params
+
+    console.log('id:', id)
+
+    const recipeLabels = await Label.findAll({
+      include: [
+        {
+          model: RecipeLabel,
+          where: { recipeId: id },
+          attributes: []
+        }
+      ]
+    })
+
+    console.log('recipeLabels:', recipeLabels)
```

Comment:
Same here, we could comment this out to quiet the noise in the server console. We could un-comment it if one of us is doing development with the parent function.